MVVMRxSwiftDemo/Sources/Presentation/Modules/SurveyQuestion/Cell/QuestionCollectionCellViewModel.swift

```diff
@@ -27,9 +28,26 @@ final class QuestionCollectionCellViewModel: QuestionCollectionCellViewModelProt
     var output: QuestionCollectionCellViewModelOutput { self }
     
     let questionText: Driver<String>
+    let answerField: Driver<AnswerField>
     
-    init(questionText: String) {
+    init(questionText: String, displayType: DisplayType, pickType: PickType, answers: [SurveyAnswer]) {
         self.questionText = Driver.just(questionText)
+        
+        switch displayType {
+        case .star, .heart, .smiley:
+            answerField = Driver.just(RatingAnswerField(displayType: displayType, answers: answers))
+        case .nps:
+            answerField = Driver.just(NPSAnswerField(answers: answers))
+        case .textarea, .textfield:
+            answerField = Driver.just(TextFieldAnswerField(displayType: displayType, answers: answers))
+        default:
+            switch pickType {
+            case .any:
+                answerField = Driver.just(MultipleChoiceAnswerField(answers: answers))
+            default:
```

Comment:
Some here, prefer to define all cases instead of using `default`
```suggestion
            case .one, .none:
```